--- stdint.h	2024-12-19 19:57:41.082512257 -0700
+++ stdint.h	2024-12-19 19:58:10.058289150 -0700
@@ -6,11 +6,6 @@
 #ifndef _KERNEL_STDINT_H_
 #define _KERNEL_STDINT_H_
 
-#ifndef KERNEL
-/* For user-space code that may include this header */
-#include_next <stdint.h>
-#else /* KERNEL */
-
 #include <machine/types.h>
 
 #if __LP64__
@@ -208,6 +203,4 @@
 #define INTMAX_C(v)  (v ## LL)
 #define UINTMAX_C(v) (v ## ULL)
 
-#endif /* KERNEL */
-
 #endif /* _KERNEL_STDINT_H_ */
